source/aggregation-pipeline-builder.txt

```diff
@@ -14,21 +14,29 @@ Aggregation Pipeline Builder
 
 *New in version 1.14.0*
 
-The Aggregation Pipeline Builder in |compass| provides the ability to
-create :manual:`aggregation pipelines </core/aggregation-pipeline/>` to
-process data. In aggregation pipelines, documents in a collection
-or view pass through stages where |compass| processes them into a
-set of aggregated results. You can change the stages and results to
-suit your needs.
-
-To start building an aggregation pipeline for a :doc:`collection or view
-</collections>`, choose the collection and click the
-:guilabel:`Aggregations` tab.
-A blank aggregation pipeline displays. The :guilabel:`Preview of
-Documents in the Collection` section of the :guilabel:`Aggregations`
-view displays 20 documents sampled from the current collection.
-
-You can:
+The Aggregation Pipeline Builder in |compass| lets you create
+:manual:`aggregation pipelines </core/aggregation-pipeline/>` to process
+data. Aggregation pipelines are made up of :ref:`stages
+</reference/operator/aggregation-pipeline/>` which process documents
+from a collection of view. For example, you can use aggregation stages
```

Comment:
Did you mean maybe `a collection or a view`?

Reply:
Yep, my mistake! Fixed.